### PR TITLE
sys-kernel/coreos-modules: disable CONFIG_NFS_V4_2_READ_PLUS

### PR DESCRIPTION
**Pull request by ader1990**
Disable CONFIG_NFS_V4_2_READ_PLUS kernel config, as Linux kernel >= 6.6 enabled the CONFIG_NFS_V4_2_READ_PLUS config option by default, and nfs-ganesha version <= 6.1 is broken due to mishandling of the read_plus operation.

See:
https://github.com/nfs-ganesha/nfs-ganesha/commit/24da5c30429bb1ee0bfde3644ab5c3b84daa778e

See: https://github.com/flatcar/Flatcar/issues/1565

See: https://github.com/nfs-ganesha/nfs-ganesha/issues/1188

## Testing done

Manually checked on a two node K8S cluster that the issue is no longer present when CONFIG_NFS_V4_2_READ_PLUS=n.

- [ ] Changelog entries added in the respective `changelog/` directory (user-facing change, bug fix, security fix, update)
- [ ] Inspected CI output for image differences: `/boot` and `/usr` size, packages, list files for any missing binaries, kernel modules, config files, kernel modules, etc.

### CODE REVIEW

**Comment by krnowak** [comment]
Do we want to reenable this CONFIG at some point?

**Comment by ader1990** [comment]
> Do we want to reenable this CONFIG at some point?

Maybe, I have not tried/found yet a nfs-ganesha version that fixes the issue, as it is not released yet, waiting for https://github.com/kubernetes-sigs/nfs-ganesha-server-and-external-provisioner/pull/152

**Review by jepio — APPROVED** [review]

File: changelog/updates/2024-10-22-disable-CONFIG_NFS_V4_2_READ_PLUS.md [hunk 1]
```diff
@@ -0,0 +1 @@
+- Disable CONFIG_NFS_V4_2_READ_PLUS kernel config to fix nfs-ganesha ([#2390](https://github.com/flatcar/scripts/pull/2390))
```
[the remark this answers is not in the document]
Comment (dongsupark) [reply changelog/updates/2024-10-22-disable-CONFIG_NFS_V4_2_READ_PLUS.md:1]: It should be located under `changelog/changes`. As it went instead under `updates`, all changelog entries are being put into the `updates` section.
I am manually fixing it.

[the remark this answers is not in the document]
Comment (dongsupark) [reply changelog/updates/2024-10-22-disable-CONFIG_NFS_V4_2_READ_PLUS.md:1]: In the future, can you please specify which repo it is in the changelog?

Not `([#2390](https://github.com/flatcar/scripts/pull/2390))`, but `([scripts#2390](https://github.com/flatcar/scripts/pull/2390))`, or `[Flatcar#1565](https://github.com/flatcar/Flatcar/issues/1565)`. Without that, it is pretty confusing for readers.

**Pull request merged**